activemodel/lib/active_model/attribute_methods.rb

```diff
@@ -234,6 +243,34 @@ def alias_attribute(new_name, old_name)
         end
       end
 
+      def compilable_alias_attribute_method_body_for(old_name, pattern, parameters) # :nodoc:
```

Comment:
Rather than no-doc these, they can be private since they aren't called outside this file.